test/tests/fvkernels/mms/skewness-correction/diffusion/skewness-correction-check.py

```diff
@@ -0,0 +1,418 @@
+import gmshparser
```

Comment:
Is this script used by anything? Given the imports I would not think so
If it s not covered by the tests it ll get deprecated

Reply:
I uploaded the python script because it is almost a one-to-one equivalent to MooseFV's FVDiffusion and it is a useful tool for debugging. I will remove it with the next commit!

Reply:
If we now reproduce the python script behavior with MOOSE, we no longer need it in the repository, but nice to have it in the commit history